assignments/function-conversion.js

```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = () => {
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
-// let anotherFunction = function (param) {
+// let anotherFunction = (param) => {
 //   return param;
 // };
 // anotherFunction("Example");
 
-// let add = function (param1, param2) {
+// let add = (param1, param2) => {
```

Comment:
Remember that you can use implicit returns with arrow functions. So instead of writing it as done above, you could also do something like this:
```js
let add = (param1, param2) => param1 + param2;
```